rental.rb

```diff
@@ -11,13 +11,4 @@ def initialize(date, person, book)
     @book = book
     book.rentals << self
   end
-
-  def to_json(*args)
```

Comment:
Please correct the spellings of Gemfile in the root directory as it does not allow to run `bundle install`. It should be `Gemfile` instead of `gemfile`.